### ISSUE TITLE
[COMPILING] Installation of funkin.vis fails with "Execution error"

### ISSUE THREAD

**Issue by EliteMasterEric** [FunkinCrew/Funkin#2298]
When attempting to build the project on Windows, some users experience the following error:

![image](https://github.com/FunkinCrew/Funkin/assets/4635334/89c18fcb-c873-4dcf-ad7c-4dca6189b860)

```
PS C:\Users\godee\Desktop\Funkin> hmm reinstall --force funkin.vis
reinstalling git library: funkin.vis from https://github.com/FunkinCrew/funkVis (ref: Some(98c9db09f0bbfedfe67a84538a5814aaef80bdea), dir: None)
haxelib --never remove funkin.vis
Error: Library funkin.vis is not installed
Warning: command "haxelib --never remove funkin.vis" failed with status: 1 in dir: C:\Users\godee\Desktop\Funkin/
haxelib --never git funkin.vis https://github.com/FunkinCrew/funkVis 98c9db09f0bbfedfe67a84538a5814aaef80bdea
Installing funkin.vis from https://github.com/FunkinCrew/funkVis branch: 98c9db09f0bbfedfe67a84538a5814aaef80bdea
Error: std@sys_remove_dir
Execution error: command "haxelib --never git funkin.vis https://github.com/FunkinCrew/funkVis 98c9db09f0bbfedfe67a84538a5814aaef80bdea" failed with status: 1 in cwd: C:\Users\godee\Desktop\Funkin/
Use "hmm help" to see usage
PS C:\Users\godee\Desktop\Funkin>
```

This issue is related to the game trying to download the code for Nene's visualizer (which is in a separate dependency library) and failing to do so.

**Comment by TheJustAnto** [comment]
So is there a way to fix this error ?

**Comment by Eviethecoder** [comment]
you could try to manually download the library through the hmm files GitHub link for it. and add it to the haxelib folder manually. at least till this issue gets solved

**Comment by Gato-chistoso** [comment]
`haxelib remove funkin.vis` then `haxelib git funkin.vis https://github.com/FunkinCrew/funkVis`

**Comment by prplnorangesoda** [comment]
Getting the same error
![image](https://github.com/FunkinCrew/Funkin/assets/60671867/41129523-da9d-4b8c-b658-b4fe96e6d48e)
Original command: `hmm install`

```
Download complete: 295.46KB in 0s (3110KB/s)
Installing format...
  Current version is now 3.5.0
Done
haxelib --never git funkin.vis https://github.com/FunkinCrew/funkVis 98c9db09f0bbfedfe67a84538a5814aaef80bdea 
Installing funkin.vis from https://github.com/FunkinCrew/funkVis branch: 98c9db09f0bbfedfe67a84538a5814aaef80bdea
Error: std@sys_remove_dir
Execution error: command "haxelib --never git funkin.vis https://github.com/FunkinCrew/funkVis 98c9db09f0bbfedfe67a84538a5814aaef80bdea" failed with status: 1 in cwd: D:\raf\Documents\.Code\funkin/
Use "hmm help" to see usage
```
Re-running `hmm install` causes it to skip over funkin.vis and installs all the other modules without error. Attempting to compile causes this error:
```
source/funkin/audio/visualize/ABotVis.hx:11: characters 8-39

 11 | import funkin.vis.dsp.SpectralAnalyzer;
    |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    | Type not found : funkin.vis.dsp.SpectralAnalyzer
``` 

Following @Gato-chistoso's comment allows me to compile with `lime test html5`

**Comment by NotHyper-474** [comment]
I think I might have figured it out, the issue seems to be the commit that hmm tries to dowload/clone from.  
Replacing the commit hash ref to the one from the latest commit and running `hmm reinstall --force funkin.vis` was successful.

**Comment by NotHyper-474** [comment]
Curiously 98c9db09f0bbfedfe67a84538a5814aaef80bdea leads to [this commit](https://github.com/FunkinCrew/funkVis/commit/98c9db09f0bbfedfe67a84538a5814aaef80bdea) with this warning: 
![image](https://github.com/FunkinCrew/Funkin/assets/40342021/e9db9672-55c9-458f-b9f2-b452aeb6cd99)

And there's another commit which appears to be the exact same but with a different hash and no warning: [d7f88b765a9e4ef51a1628d53354abfdea8b928d](https://github.com/FunkinCrew/funkVis/commit/d7f88b765a9e4ef51a1628d53354abfdea8b928d)

**Comment by NotHyper-474** [comment]
This issue is fixed in develop branch. For anyone still having this issue switch to that branch and run ``hmm reinstall --force funkin.vis``

**Comment by EliteMasterEric** [comment]
This should be fixed in v0.3.3.
